web/packages/design/src/Alert/Alert.tsx

```diff
@@ -379,11 +385,21 @@ export const ActionButton = ({
       fill={fill}
       intent={intent}
       onClick={onClick}
+      inputAlignment={inputAlignment}
+      disabled={disabled}
+      title={title}
```

Comment:
I should've probably used `HoverTooltip` for this instead of `title`, I think the design team doesn't like using `title`.

Reply:
I like `title`, our tooltip it too large to me